### PR TITLE
refactor: Rewrite `Interoperability` to include return types of imports 

### LINKED ISSUE #63: Rewrite `Interoperability` to include return types of imports
[This sentence](https://github.com/flix/programming-flix/blob/6864d39b90922b78f8d3e5be7ae26b915e367166/src/content/Interoperability.js#L84) is out-of-date, since we know always specify return types of java imports.

The `Interoperability` page should be updated to let the user know to specify the return type and effect of imports.

In [the summary](https://github.com/flix/programming-flix/blob/6864d39b90922b78f8d3e5be7ae26b915e367166/src/content/Interoperability.js#L180) the return types are not included either.

### PR DESCRIPTION
**Pull request by ninaandrup**
Closes #63

### CODE REVIEW

**Comment by ninaandrup** [comment]
@magnus-madsen Do you want me to include return types in [the table at the bottom of the page](https://github.com/flix/programming-flix/blob/6864d39b90922b78f8d3e5be7ae26b915e367166/src/content/Interoperability.js#L219)? I am not sure, if it going to be too messy. But it does serve the purpose of giving the user an overview of imports, and the return types is mandatory now.

**Comment by magnus-madsen** [comment]
> @magnus-madsen Do you want me to include return types in [the table at the bottom of the page](https://github.com/flix/programming-flix/blob/6864d39b90922b78f8d3e5be7ae26b915e367166/src/content/Interoperability.js#L219)? I am not sure, if it going to be too messy. But it does serve the purpose of giving the user an overview of imports, and the return types is mandatory now.

You decide. Either is fine with me.

File: src/content/Interoperability.js [hunk 1]
```diff
@@ -219,7 +227,11 @@ exists(f)`}
                 <SubSection name="Summary">
 
                     <p>
-                        The table below gives an overview of the syntax:
+                        The table below gives an overview of the syntax.
+                    </p>
+
+                    <p>
+                        Note: the return types and effects must always be specifed but are omitted for a simpler overview.
```
Comment (ninaandrup) [inline src/content/Interoperability.js:234]: I opted for a simple note above the overview table for imports.

Comment (magnus-madsen) [reply src/content/Interoperability.js:234]: 👍

**Pull request merged**